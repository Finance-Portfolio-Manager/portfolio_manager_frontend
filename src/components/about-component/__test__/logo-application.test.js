import React, { Component } from 'react';
import ReactDOM from 'react-dom';
import LogoCard from './../logo-application';

import { render, cleanup } from '@testing-library/react';
import "@testing-library/jest-dom/extend-expect";

import renderer from "react-test-renderer";

import { shallow, ShallowWrapper, mount } from "enzyme";

import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });
const wrapper = shallow(<LogoCard></LogoCard>);

test("matches snapshot", ()=>{
    const tree = renderer.create(<LogoCard></LogoCard>).toJSON();
    expect(tree).toMatchSnapshot();
})

test("logo renders", ()=>{
    expect(wrapper.find("Logo").length).toBe(1);
})

test("Card text renders", ()=>{
    expect(wrapper.find("CardText").text()).toContain("APEX STOCKS");
})