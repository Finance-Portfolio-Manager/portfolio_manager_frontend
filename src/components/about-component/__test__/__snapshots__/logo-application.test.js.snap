// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="border border-0 logoApp m-3 accent-text card"
  id="logo-card"
>
  <div
    className="card-body"
    id="logo"
  >
    <svg
      viewBox="0 0 12.93 11.56"
      xmlns="http://www.w3.org/2000/svg"
    >
      <g
        fill="rgb(59, 101, 255)"
      >
        <path
          d="M7.15,0A5.78,5.78,0,0,0,1.37,5.78a5.66,5.66,0,0,0,.51,2.36L0,10.2l.87.8L2.51,9.21A5.78,5.78,0,1,0,7.15,0Zm0,10.38A4.55,4.55,0,0,1,3.33,8.31l.49-.53L5,9.07l3.8-4.3,1.84,2,.87-.8L8.78,3,5,7.31,3.82,6l-1,1.14a4.7,4.7,0,0,1-.22-1.39,4.6,4.6,0,1,1,4.6,4.6Z"
        />
      </g>
    </svg>
    <p
      className="d-flex justify-content-center card-text"
      id="appName"
    >
      APEX STOCKS
    </p>
  </div>
</div>
`;
